UKHomeOffice/sgmr-service
# NMSW-98 Remove AIS number from the Pleasure Craft form

### AC / Description of changes

GIVEN I am on the spcr pleasure craft form
WHEN I view the yes/no questions
THEN I no longer see the AIS number question

### To Test

- run app locally
- go to localhost:8080
- sign in
- Navigate to /pleasure-crafts
- Create a new PC
- fill in page 1
- > On page 2 you should not see a question about AIS
- Fill out the rest of the form and save the please craft
- > You should not get any errors

### Notes

- Cannot check whether cypress tests are correct as cannot run them locally, these will have to be tested on dev

---

**Remember to merge to feature branches not `master`.**
